examples/agent_feature_store/agent.py

```
@@ -0,0 +1,816 @@
+
+def _call_llm(messages: list, use_tools: bool = True) -> dict:
+    """Make a single LLM API call, returning the parsed choice dict."""
+    url = f"{OPENAI_BASE_URL}/chat/completions"
+    payload: dict[str, Any] = {
+        "model": LLM_MODEL,
+        "messages": messages,
+        "temperature": 0.3,
+    }
+    if use_tools:
+        payload["tools"] = TOOLS_SPEC
+    resp = requests.post(
```

Ruff (0.15.9)

[error] Probable use of `requests` call without timeout

(S113)